hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/hudi/streaming/HoodieSourceOffset.scala

```diff
@@ -45,17 +45,21 @@ case class HoodieSourceOffset(commitTime: String) extends Offset {
 
 
 object HoodieSourceOffset {
-  val mapper = new ObjectMapper with ScalaObjectMapper
-  mapper.setSerializationInclusion(Include.NON_ABSENT)
-  mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
-  mapper.registerModule(DefaultScalaModule)
+
+  lazy val mapper: ObjectMapper = {
```

Comment:
here we just need to remove the `with ScalaObjectMapper` in line#48 before?

Reply:
Yes, as `scalaObjectMapper` we can replaced with `ClassTag`, we can safely and simply solve the conflict issue with different scala version(2.11 and 2.12, etc)